idrstream/DP_idr.py

```diff
@@ -290,90 +293,21 @@ def compile_DP_batch_index_csv(self, batch_metadata: pd.DataFrame):
                 index_csv_data.append(image_data)
         index_csv_data = pd.DataFrame(index_csv_data)
         index_csv_data.to_csv(index_csv_save_path, index=False)
-        self.logger.info(f"Compiled index.csv file to {index_csv_save_path}")
+        # How would logging work?
+        # self.logger.info(f"Compiled index.csv file to {index_csv_save_path}")
 
-    def profile_batch(self):
+    def profile_batch_with_DP(self):
         """
         profile batch with DeepProfiler
         """
         command = f"python3 -m deepprofiler --gpu 0 --root {self.DP_project_path} --config {self.config_name} profile"
         os.system(command)
-        self.logger.info("Deep Profiler run done")
-
-    def clear_batch(self):
-        """
-        remove all intermediate files that are unecessary for next batch to run
-        """
-        images = pathlib.Path(f"{self.DP_project_path}/inputs/images/")
-        locations = pathlib.Path(f"{self.DP_project_path}/inputs/locations/")
-        metadata = pathlib.Path(f"{self.DP_project_path}/inputs/metadata/")
-        features = pathlib.Path(f"{self.DP_project_path}/outputs/results/features/")
-        paths_to_remove = [images, locations, metadata, features]
-
-        for path in paths_to_remove:
-            shutil.rmtree(path)
-
-        self.logger.info("Temporary batch files cleared")
+        # How would logging work?
+        # self.logger.info("Deep Profiler run done")
```

Comment:
Think about adjusting the comments here as appropriate.

Reply:
LOL I forgot I was having a convo with myself in the code. Fixed!